util/imagetools/loader.go

```diff
@@ -275,17 +276,27 @@ func (l *loader) scanSBOM(ctx context.Context, fetcher remotes.Fetcher, r *resul
 				if err != nil {
 					return err
 				}
-				as.sbom = &sbomStub{
-					SPDX: dt,
+				var spdx struct {
+					Predicate interface{} `json:"predicate"`
```

Comment:
Bit concerned that there isn't any way to extract the actual attestation. But I guess we can add some special case/command for that later.